File: client/web/sudoo/lib/domain/model/discovery/order_product_info.dart
import 'package:json_annotation/json_annotation.dart';

part 'order_product_info.g.dart';

@JsonSerializable()
class OrderProductInfo {
  final String productId;
  final String supplierId;
  final String sku;
  final String name;
  final String brand;
  final double price;
  final double weight, height, length, width;
  final List<String> images;

  OrderProductInfo(
    this.productId,
    this.supplierId,
    this.sku,
    this.name,
    this.brand,
    this.price,
    this.weight,
    this.height,
    this.length,
    this.width,
    this.images,
  );

  factory OrderProductInfo.fromJson(Map<String, dynamic> json) => _$OrderProductInfoFromJson(json);

  Map<String, dynamic> toJson() => _$OrderProductInfoToJson(this);
}

File: client/mobile/ShipperDemo/app/src/main/java/com/sudoo/shipment/model/OrderProductInfo.kt
package com.sudoo.shipment.model

data class OrderProductInfo(
    val productId: String,
    val supplierId: String,
    val name: String,
    val sku: String,
    val images: List<String>,
    val price: Double,
    val brand: String,
    val weight: Int,
    val height: Int,
    val length: Int,
    val width: Int,
)